packages/react/README.md

```diff
@@ -0,0 +1,11 @@
+# [Fluent UI React](https://dev.microsoft.com/fabric)
+
+**The React-based front-end framework for building experiences for M365.**
```

Comment:
Probably do this? M365 is a bit jargon-y. :) But this new version is much better overall!
```suggestion
**The React-based front-end framework for building experiences for Microsoft 365.**
```